src/styles/typography.scss

```diff
@@ -57,6 +57,12 @@ h3 {
   color: $rocket;
 }
 
+a {
+  &:hover {
+    color: $rocket;
```

Comment:
for consistency, would it be useful to:

* offer `@extend`-able classes to hover for `$rocket` and $white`
* add `transition` to this hoverable class (since after the gentle hover of `Button`, the lack of transition on links is a tad jarring - this transition can probably be faster than the button one)

Reply:
I don't get what you mean by the first point. What's the reason to use extendable class here?

Reply:
You added a similar on-hover behaviour for the nav, might not be the last time we use it - might not be necessary though

I do still think this hover could use a transition, whether or not you add an extendible class 👍